src/main/resources/META-INF/plugin.xml

```diff
@@ -52,6 +52,9 @@
 
         <applicationService
                 serviceImplementation="com.sourcegraph.cody.config.CodyPersistentAccounts"/>
+
+        <actionPromoter implementation="com.sourcegraph.cody.edit.CodyActionPromoter"/>
```

Comment:
this is the actual fix for this one - thank you JetBrains community Slack for help 😃